Source/Core/Core/CheatSearch.cpp

```diff
@@ -0,0 +1,602 @@
+// Copyright 2021 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
```

Comment:
This goes for all files, but you should use SPDX tags instead:
```suggestion
// SPDX-License-Identifier: GPL-2.0-or-later
```